README.rst

```diff
@@ -23,63 +23,246 @@ Example
+   every operation - in this case the name of the asset is 'radiation bag 1'.
+
+   Arguments to the archivist are ususally strings - in this example radioactive is 
+   'true' which archivist will treat as a boolesn.
+
+
+.. code-block:: yaml
+
+    ---
+    # Demonstration of applying a Richness compliance policy to an asset that undergoes
+    # events that may or may not make the asset compliant or non-compliant.
+    #
+    # The operation field is a string that represents the method bound to an endpoint and
+    # the args and kwargs correspond to the arguments to such a method.
+    #
+    # NB the assets and events endpoints require all values to be strings. Other values may
+    # be of the correct type such as confirm which is a boolean.
     #
-    # store asset on the DLT or not. If DLT is not enabled for the user an error will occur if
-    # ProofMechanism.KHIPU is specified. If unspecified then SIMPLE_HASH is used
-    # i.e. not stored on the DLT...
-    # proof_mechanism = ProofMechanism.KHIPU
-    props = {
-        "proof_mechanism": ProofMechanism.KHIPU.name,
-    }
-
-    # The first argument is the properties of the asset
-    # The second argument is the attributes of the asset
-    # The third argument is wait for confirmation:
-    #   If @confirm@ is True then this function will not
-    #   return until the asset is confirmed on the blockchain and ready
-    #   to accept events (or an error occurs)
-    #   After an asset is submitted to the blockchain (submitted),
-    #   it will be in the "Pending" status.
-    #   Once it is added to the blockchain, the status will be changed to "Confirmed"
-    try:
-        asset = arch.assets.create(props=props, attrs=attrs, confirm=True)
-    except Archivisterror as ex:
-        print("error", ex)
-    else:
-        print("asset", asset)
-    
+    # For example kwargs.attrs.radioactive is a string - internally archivist will see this
+    # as a boolean.
+    operations:
+
+      # note the values to the assets.create method are string representations of boolean
+      # and numbers
+      - operation: assets.create
+        to_print: Create an empty radiation bag with id 1.
+        kwargs:
+          attrs:
+            arc_display_name: radiation bag 1
+            radioactive: "true"
+            radiation_level: "0"
+            weight: "0"
+          confirm: true
+
+      # setup the radiation bags to have a varing amount of radiactive waste
+      # note the values to the events.create method are string representations of boolean
+      # and numbers
+      - operation: events.create
+        to_print: Create Event adding 3 rads of radiation to bag 1, increasing its weight by 1kg.
+        asset_name: radiation bag 1
+        args:
+          - operation: Record
+            behaviour: RecordEvidence
+          - arc_description: add waste to bag
+            arc_evidence: see attached report
```

Comment:
This only makes sense if there are attachments on the Event. Please either add attachments to this example or alter thee evidence string to something more appropriate.

Reply:
I have used the example in docs.rkvst.com for standard RecordEvidence event.